server/views/pages/404.njk

```diff
@@ -0,0 +1,9 @@
+{% extends "../layouts/layout.njk" %}
+
+{% set pageTitle = applicationName + " - 404 error" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">This page cannot be found</h1>
+  <p class="govuk-body">Return to <a href="/pop?bypass=true">Home</a></p>
```

Comment:
I'm not sure we should give the impression that they will go to the home page if they were never logged in? The bypass is ideally for manual use only.